### PR TITLE
Install Vue via CLI and enable PWA

### LINKED ISSUE #2: Create VueJS Frontend
- [x] Init project using Vue CLI
- [x] Enable PWA support and verify PWA prod build works via Chrome Lighthouse
- [x] Should live under `frontend`
- [x] Add concise dev info to `frontend/README.md` 
- [x] Add Wiki page for verbose Frontend dev info with links to guides, docs, and info on how Vue was installed

### PR DESCRIPTION
**Pull request by allisonsierra**
- resolves #2
- Init project using Vue CLI
- Enable PWA support
- Verified PWA prod build works via Chrome Lighthouse

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.22.3",
+    "register-service-worker": "^1.7.2",
+    "vue": "^3.2.33",
```
[the remark this answers is not in the document]
Comment (allisonsierra) [reply frontend/package.json:13]: @sonatype-lift ignore

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.8.3",
+    "register-service-worker": "^1.7.2",
+    "vue": "^3.2.13",
```
[the remark this answers is not in the document]
Comment (allisonsierra) [reply frontend/package.json]: @sonatype-lift ignore

File: frontend/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (allisonsierra) [reply frontend/package.json:13]: @sonatype-lift ignore

**Comment by allisonsierra** [comment]
Running `npx eslint --config=.eslintrc.js --format=json -o ./eslint-out/report.json .eslintrc.js babel.config.js vue.config.js` from the `frontend` directory works properly. `npm run lint` works as well. This seems like a Lift bug.

The CVE it is calling out seems outdated and doesn't exist on Sonatype's website so I'm ignoring it. Updated to latest version of Vue and all other deps.

https://my.sonatype.com/nexus-intelligence/sonatype-2019-0454

Ready for a squash merge after review.

**Review by amy-keibler — APPROVED** [review]

**Pull request merged**